docs/design/diagrams/managed_subscribe_interceptor_sequence.puml

```diff
@@ -1,8 +1,10 @@
 @startuml Managed Subscribe Interceptor
 title Managed Subscribe Interceptor Sequence
 participant "Digital Twin Provider" as DTProvider
+box "Ibeji"
 participant "Managed Subscribe Module" as ManagedSubscribeModule
```

Comment:
I feel that you should have abstraction parity amongst all of your actors inside of the Ibeji border.  You use module for Managed Subscribe and service for fundamental Ibeji functionality.

To fix this, could you rename "Managed Subscribe Module" to "Managed Subscribe Interceptor"?  Alternatively, you could rename "InVehicle Digital Twin" to "Fundamental Ibeji Module" (or something like that).

Reply:
Devin and I chatted about this.  I'm fine with how it is today.  Devin will later provide a sequence diagram that shows what is happening inside the Managed Subscribe Module during this use case.